.github/workflows/check-links-in-prod.yaml

```diff
@@ -3,93 +3,139 @@
 name: Check links
 
 on:
-  workflow_dispatch:
+  workflow_dispatch: {}
+  schedule:
+    - cron: '39 0 20 * *' # every 20th of the month at 00:39
 
 jobs:
   check-links:
     runs-on: ubuntu-latest
     steps:
-      - name: Pull images
+      - name: Pull image
         run: docker pull ghcr.io/linkchecker/linkchecker:latest
 
       - name: check-links-in-overview-pages
         run: |
           docker run --rm --name linkchecker \
+            --volume ${PWD}:/workdir --workdir /workdir \
             ghcr.io/linkchecker/linkchecker:latest \
             https://docs.giantswarm.io/overview/ \
             --check-extern \
-            -t 1 -r 2 || echo "failed1=true" >> "$GITHUB_ENV"
+            --threads 1 \
+            --recursion-level 2 \
+            --no-status \
+            --file-output html/utf8/overview.html \
```

Comment:
`html/utf8/overview.html` is the very stupid format of `TYPE/ENCODING/FILENAME`.